script/install-bazel

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+set -e
+
+# linux install only
+test "Darwin" = "$(uname)" && exit 1
```

Comment:
With `set -e` this may terminate the script.

Reply:
Oh I see, so the exit is redundant?

Reply:
Nope, I though that the failure of the first command in the `&&` list would give the entire list a failed exit code and cause the script to abort after it. But [TIL](http://pubs.opengroup.org/onlinepubs/9699919799/utilities/V3_chap02.html#tag_18_25) that `The -e setting shall be ignored when executing... ...any command of an AND-OR list other than the last`.